contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -733,7 +733,7 @@ open class ContourLayout(
     internal fun height(): YInt = y.range().toYInt()
 
     internal fun preferredWidth(): XInt {
-      return if (view.visibility == View.GONE) {
+      return if (view.visibility == GONE) {
```

Comment:
Inline refactoring also removed `View` class reference since `ContourLayout` is a `View`. Should be ok I guess.